### DISCUSSION TITLE
Allow for other privacy preserving Analytics Platforms e.g. (Plausible)

### DISCUSSION THREAD

**Discussion by alexgallacher** [forem/forem#18552] [Feature Requests]
Hi All,

Is there any planned support or can there be a FR for support for privacy preserving analytics such as [Plausible](https://plausible.io/)? 

GA4 is terrible from a privacy lense!

Cheers.

**Comment by maestromac** [comment]
No plan at the moment but thank you for voicing your tool of choice.
